upgrade-2.x.sh

```diff
@@ -45,6 +47,9 @@ Options:
         If not defined, then the update will try to run for the HOSTOS_VERSION's
         original supervisor release.
 
+    -n, --nolog
```

Comment:
I would rename the flag to be clear that it only deactivates log to file.

Reply:
@agherzan very much agree, this form is to keep it the same as the 1.x->1.x flags, which surprised me with this slightly off meaning (see https://github.com/resin-os/meta-resin/blob/1.X/meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh#L68-L69 )

So I wonder, is it better to take this chance to break uniformity between scripts, or not?

Reply:
Uh I see. OK. Let's leave it as it is for now.